package.json

```diff
@@ -18,86 +18,127 @@
     "Rômulo Machado <romulo.machado5@gmail.com>",
     "Robbie Wagner <rwwagner90@gmail.com>"
   ],
-  "directories": {
-    "doc": "doc",
```

Comment:
do we still have a doc folder? do we still need this directories block?

Reply:
This is just part of the blueprints, I think? If we do not want it, we should probably alter the blueprints.

Reply:
well I'm actually commenting on the removal here not the addition. We need the `#src` thing to make tests work properly, I just wonder if we are missing the thing that was deleted here

Reply:
Ohhh, sorry, I was confused. I'm not sure.